src/nimbledroid/config.js

```diff
@@ -1,5 +1,29 @@
-import CONFIG from './config';
-import { Log } from '../../vendor/logs';
+import { Log } from '../vendor/logs';
+
+export const CONFIG = {
+  packageIdLabels: {
+    'org.mozilla.klar': 'Firefox Klar',
+    'com.chrome.beta': 'Chrome Beta',
+    'org.mozilla.geckoview_example': 'GeckoView Example',
+    'org.mozilla.reference.browser': 'Reference Browser',
+    'org.mozilla.fenix': 'Firefox Preview',
+  },
+  products: [
+    'org.mozilla.klar',
+    'com.chrome.beta',
+    'org.mozilla.geckoview_example',
+    'org.mozilla.reference.browser',
+    'org.mozilla.fenix',
+  ],
+  summaryTable: [
+    'org.mozilla.klar',
+    'org.mozilla.geckoview_example',
+    'com.chrome.beta',
+  ],
+  baseProduct: 'org.mozilla.geckoview_example',
+  compareProduct: 'com.chrome.beta',
+  targetRatio: 1.2,
+};
```

Comment:
the above is from https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/414/files#diff-12464aba5618f7693c828c892396a8db